src/NUnitFramework/framework/Assume.cs

```diff
@@ -34,7 +34,7 @@ namespace NUnit.Framework
     /// result. If an assumption is not met, the test
     /// should produce an inconclusive result.
     /// </summary>
-    public class Assume
+    public abstract class Assume
```

Comment:
This would be `static` (= `sealed abstract`) but we want to allow inheritance in these particular classes as an extensibility mechanism since C# does not allow static extension members.